packages/api/lib/utils.js

```diff
@@ -36,21 +36,6 @@ function parseException(exception) {
   };
 }
 
-/**
- * Returns the name and version of a collection based on
- * the collectionId used in elasticsearch indexing
- *
- * @param {string} collectionId - collectionId used in elasticsearch index
- * @returns {Object} name and version as object
- */
-function deconstructCollectionId(collectionId) {
```

Comment:
This was already in both places? here and @cumulus/message/Collections ?

Reply:
It was in both places.